<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bot Falhas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(85, 110, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titulo">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>AN√ÅLISE DE FALHAS</string>
     </property>
    </widget>
    <widget class="QLabel" name="foto_robo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/imagem-robo.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="arquivo_input">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_selecionar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ccc;</string>
     </property>
     <property name="text">
      <string>Selecionar arquivo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_analisar">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ccc;
</string>
     </property>
     <property name="text">
      <string>Analisar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabela_resumo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
border-radius: 15px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_imprimir">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ccc;</string>
     </property>
     <property name="text">
      <string>Imprimir</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
